include/deal.II/fe/fe.h

```diff
@@ -1947,7 +1948,9 @@ class FiniteElement : public Subscriptor,
    * FETools::compute_node_matrix() function.
    *
    * In more detail, let us assume that the generalized support points
-   * (see @ref GlossGeneralizedSupport "this glossary entry") of the current
+   * (see
+   * @ref GlossGeneralizedSupport "this glossary entry"
+   * ) of the current
```

Comment:
This is probably going to look funny with a gratuitous space, but not worth worrying about.